presqt/api_v1/utilities/validation/file_duplicate_action_validation.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import status
+
+from presqt.exceptions import PresQTValidationError
+
+
+def file_duplicate_action_validation(request):
```

Comment:
I really like how you are breaking things down into smaller functions.
Really helps with the cognitive load.